### PR TITLE
feat: refresh content for 2020 Q3

### PR DESCRIPTION
**Pull request by lidel**
This PR:

-  updates existing content by adding references to recent wins in DWeb adoption
-  moves  URIs and Origin to the top, as we had the most wins in those areas
-  adds Opera because they support IPFS URIs in their Android version
- updates `content/programmable-custom-protocol-handlers.md` to reduce confusion when reader opens it via direct link included in https://blog.chromium.org/2020/09/chrome-86-improved-focus-highlighting.html
- updates browser logos to latest versions (where possible)

### CODE REVIEW

File: content/programmable-custom-protocol-handlers.md [hunk 1]
```diff
@@ -1,26 +1,30 @@
...
+- Firefox: Supportive: [bug #1271553](https://bugzilla.mozilla.org/show_bug.cgi?id=1271553); In the past prototyped WebExtension API as a part of [libdweb](https://github.com/mozilla/libdweb): [Protocol handler API](https://github.com/mozilla/libdweb/issues/2)
 - Safari: ?
 
 ## Motivation
 
-Users of a decentralized web want to get content from a network of peers rather than a specific server.
+Users of the distributed web want to get content from a network of peers rather than a specific server.
+
+If the browser vendor does not support distributed web schemes natively,
+there should be an api that allows a WebExtension to register itself as **the handler for new protocols**, and can return responses to the browser.
```
Comment (jessicaschilling) [inline content/programmable-custom-protocol-handlers.md:21]:
```suggestion
there should be an API that allows a WebExtension to register itself as **the handler for new protocols**, and can return responses to the browser.
```

File: content/programmable-custom-protocol-handlers.md [hunk 2]
```diff
@@ -1,26 +1,30 @@
...
 
 Right now, only possible to register custom protocols handlers statically, from the `manifest.json`, where a uri is mapped to a url template, and a separate, centralised service must be maintained to handler the mapping. The custom protocol handler can only redirect the user to an (http) url, it cannot return content directly to the browser. The redirect is visible in the browsers url bar, and the original uri is no longer visible to the user.
 
-## Usage Documentation
+## Need: more powerful API for browser extensions
```
Comment (jessicaschilling) [inline content/programmable-custom-protocol-handlers.md:25]:
```suggestion
## Need: More powerful API for browser extensions
```

File: content/programmable-custom-protocol-handlers.md [hunk 3]
```diff
@@ -40,12 +44,16 @@ browser.protocol.registerStreamProtocol('ipfs', (request, callback) => {
 
 This example shows how you would add a handler for the `ipfs` protocol and have it return a file retrieved via the IPFS network as a stream to the browser.
 
+This is just a mockup based on prior art from Electron. The new API should leverage latest JS features, such as async iterators.
```
Comment (jessicaschilling) [inline content/programmable-custom-protocol-handlers.md:47]:
```suggestion
This is just a mockup based on prior art from Electron. The new API should leverage the latest JS features, such as async iterators.
```

File: data/topics.json [hunk 4]
```diff
@@ -8,7 +8,11 @@
     },
     "brave": {
       "level": 1,
-      "details": "Supportive: [brave-browser/issues/819](https://github.com/brave/brave-browser/issues/819)"
+      "details": "One-click install of IPFS in production. Work in progress on native`ipfs://` URI backed by either a public gateway or embedded go-ipfs support: [brave-browser/issues/10220](https://github.com/brave/brave-browser/issues/10220)"
```
Comment (jessicaschilling) [inline data/topics.json:11]:
```suggestion
      "details": "One-click install of IPFS in production. Work in progress on native `ipfs://` URI backed by either a public gateway or embedded go-ipfs support: [brave-browser/issues/10220](https://github.com/brave/brave-browser/issues/10220)"
```

File: data/topics.json [hunk 5]
```diff
@@ -26,37 +30,52 @@
...
   {
-    "name": "WebRTC issues",
-    "description": "Committed to fixing WebRTC bugs and performance issues",
+    "name": "Native or programmable custom protocol handlers",
+    "description": "Vendors who either support Distributed Web Schemes natively, or allow a browser extension to register itself as the handler for new protocols.",
+    "doc": "programmable-custom-protocol-handlers",
     "beaker": {
-      "level": 0
+      "level": 2,
+      "details": "Built in support for `dat://` and `hyper://` URIs"
```
Comment (jessicaschilling) [inline data/topics.json:38]:
```suggestion
      "details": "Built-in support for `dat://` and `hyper://` URIs"
```

File: data/topics.json [hunk 6]
```diff
@@ -26,37 +30,52 @@
...
     },
     "safari": {
       "level": 0
     }
   },
 
   {
-    "name": "Local Discovery",
-    "description": "Allow a WebExtension to do DNS-Based Service Discovery API as per [RFC6763](https://tools.ietf.org/html/rfc6763).",
+    "name": "Origin isolation and Secure Context",
```
Comment (jessicaschilling) [inline data/topics.json:66]:
```suggestion
    "name": "Origin Isolation and Secure Context",
```

File: data/topics.json [hunk 7]
```diff
@@ -65,48 +84,50 @@
...
       "level": 0
     }
   },
 
   {
-    "name": "Programmable custom protocol handlers",
-    "description": "Allow a WebExtension to register itself as the handler for new protocols, and can return responses to the browser.",
-    "doc": "programmable-custom-protocol-handlers",
+    "name": "Local Discovery",
+    "description": "Allow a WebExtension to do DNS-Based Service Discovery API as per [RFC6763](https://tools.ietf.org/html/rfc6763).",
```
Comment (jessicaschilling) [inline data/topics.json:122]:
```suggestion
    "description": "Allow a WebExtension to do DNS-based Service Discovery API as per [RFC6763](https://tools.ietf.org/html/rfc6763).",
```

File: data/topics.json [hunk 8]
```diff
@@ -116,31 +137,34 @@
...
     "chrome": {
       "level": 0
     },
     "edge": {
       "level": 0
     },
     "firefox": {
-      "level": 0
+      "level": 1,
+      "details": "Supportive: [bug #1247628](https://bugzilla.mozilla.org/show_bug.cgi?id=1247628)<br/>Prototyping API as a part of [libdweb](https://github.com/mozilla/libdweb): [UDP Socket](https://github.com/mozilla/libdweb/issues/4), [TCP Socket](https://github.com/mozilla/libdweb/issues/5). On the other hand [negative position](https://github.com/mozilla/standards-positions/issues/431#event-3756882378) regarding raw sockets in regular Web API."
```
Comment (jessicaschilling) [inline data/topics.json:167]:
```suggestion
      "details": "Supportive: [bug #1247628](https://bugzilla.mozilla.org/show_bug.cgi?id=1247628)<br/>Prototyping API as a part of [libdweb](https://github.com/mozilla/libdweb): [UDP Socket](https://github.com/mozilla/libdweb/issues/4), [TCP Socket](https://github.com/mozilla/libdweb/issues/5). On the other hand, [negative position](https://github.com/mozilla/standards-positions/issues/431#event-3756882378) regarding raw sockets in regular Web API."
```

**Review by jessicaschilling — CHANGES_REQUESTED** [review]
Thank you so much for doing this! Made some copy edits but otherwise LGTM.

File: data/topics.json [hunk 9]
```diff
@@ -26,37 +30,52 @@
...
     },
     "edge": {
-      "level": 0
+      "level": 1,
+      "details": "Supportive, looking into protocol handlers as part of their PWA work"
     },
     "firefox": {
       "level": 1,
-      "details": "Supportive"
+      "details": "Supportive: [bug #1271553](https://bugzilla.mozilla.org/show_bug.cgi?id=1271553)<br/>Prototyping API as a part of [libdweb](https://github.com/mozilla/libdweb): [Protocol handler API](https://github.com/mozilla/libdweb/issues/2)"
```
Comment (autonome) [inline data/topics.json:58]: "Supportive" is questionable. Libdweb has no signal of support in some time now, and no action really on protocol handler part.

**Review by autonome — APPROVED** [review]
👍🏼 thanks for doing this update!

File: data/topics.json (hunk 9 above)
Comment (lidel) [reply data/topics.json]: Good point. I've updated the score and rewrote Firefox details – mind taking a quick look?

- c9c4f76 
- 5b6c190

Comment (autonome) [reply data/topics.json]: 👍🏼

**Pull request merged**